canary/environments/pull-environments.sh

```diff
@@ -2,8 +2,9 @@
 set -e
 IFS='|'
 
-# In development, AWS_PROFILE should be set. In CI, it's not.
-[ "$AWS_PROFILE" ] && useProfile="true" || useProfile="false";
+# In development, AWS_PROFILE will be set explicitly. In CI,
+# we set it explicitly to default.
+[ -z "$AWS_PROFILE" ] && AWS_PROFILE="default"
```

Comment:
Changed the script to always use AWS profile, as per issue mentioned in https://github.com/aws-amplify/amplify-ui/pull/3498/files#r1125184155.